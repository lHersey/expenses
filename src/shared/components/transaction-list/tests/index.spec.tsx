import { render } from '@testing-library/react-native';
import Providers from 'shared/providers/test-providers';

import TransactionList from '../index';

describe('<TransactionList />', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      expect.hasAssertions();
      const rendered = render(
        <Providers>
          <TransactionList />
        </Providers>,
      ).toJSON();

      expect(rendered).toMatchSnapshot();
    });
  });
});
