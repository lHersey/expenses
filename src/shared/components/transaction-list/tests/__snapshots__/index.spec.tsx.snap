// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionList /> snapshots should match snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "display": "flex",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "marginTop": 40,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginBottom": 20,
            "paddingBottom": 0,
            "paddingLeft": 30,
            "paddingRight": 30,
            "paddingTop": 0,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#5e5e5e",
              "fontFamily": "Inter_500",
              "fontSize": 16,
            },
            Object {
              "fontFamily": "Inter_700",
              "fontSize": 20,
            },
          ]
        }
      >
        Transactions
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#5e5e5e",
              "fontFamily": "Inter_500",
              "fontSize": 16,
            },
            Object {
              "color": "#b0b0b0",
              "fontSize": 15,
            },
          ]
        }
      >
        View All
      </Text>
    </View>
    <RCTScrollView
      ItemSeparatorComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "displayName": "Styled(View)",
          "inlineStyle": InlineStyle {
            "rules": Array [
              "
  margin: 10px 0;
",
            ],
          },
          "render": [Function],
          "shouldForwardProp": undefined,
          "styledComponentId": "StyledNativeComponent",
          "target": [Function],
          "withComponent": [Function],
        }
      }
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      style={
        Array [
          Object {
            "paddingBottom": 0,
            "paddingLeft": 30,
            "paddingRight": 30,
            "paddingTop": 0,
          },
        ]
      }
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
</RNCSafeAreaProvider>
`;
